lib/lightning/accounts.ex

```diff
@@ -24,33 +27,15 @@ defmodule Lightning.Accounts do
       # coveralls-ignore-stop
     end)
 
-    # Remove credentials
+    # Remove user from projects
+    Ecto.assoc(%User{id: id}, :project_users) |> Repo.delete_all()
+
+    # Delete the credentials of the user.
+    # Note that there's a nilify constraint that set all project_credentials associated to this user to nil
     Credentials.list_credentials_for_user(id)
     |> Enum.each(&Credentials.delete_credential/1)
 
-    # Revoke access to projects
-    Projects.get_projects_for_user(%User{id: id})
-    |> Repo.preload(:project_users)
-    |> Enum.each(fn p ->
-      Projects.update_project(
-        p,
-        %{
-          "project_users" => %{
-            "0" => %{
-              "delete" => "true",
-              "user_id" => id,
-              "id" =>
-                Enum.find(p.project_users, fn pu -> pu.user_id == id end)
-                |> Map.get(:id)
-            }
-          }
-        }
-      )
-    end)
```

Comment:
Just double checking here, this isn't needed anymore because deleting the user deletes project_user records as well right?